Automattic/woocommerce-payments
# Transdaction Details: Payment Details Card

Fixes #100

ToDo:

- [x] Format all values (translate, date format, country names instead of codes, etc.)
- [x] Style accordingly
- [x] Figure out what to display while loading

#### Changes proposed in this Pull Request

Display all necessary details about the used payment method.

#### Testing instructions

1. Navigate to the details about a transaction.
2. Make sure the card details look similar to this:

![image](https://user-images.githubusercontent.com/5311119/75058619-00bec380-54e4-11ea-8c8e-0db785fb5b96.png)

Noteworthy:

- "Visa credit card" is dynamically generated, make sure the displayed value makes sense.
- As per the latest Figma designs, there are no icons next to the "check" fields.
- The address should have the correct format for the given country.

-------------------
